app/services/MatcherPool.scala

```diff
@@ -237,7 +241,11 @@ class MatcherPool(
         .check(MatcherRequest(blocksWithSkippedRangesRemoved))
         .map { matches => matches.map(_.mapMatchThroughBlocks(blocks)) }
 
-      futures.timeout(checkTimeoutDuration)(eventuallyCheck)
+      val taskName = s"MatcherPool.runMatchersForJob (type: ${matcher.getType}, categories: ${matcher.getCategories.map(_.name).mkString(", ")})"
```

Comment:
Would it be useful to add the matcher type and category as markers?

Reply:
Great point!